templates/rhsm.conf.erb

```diff
@@ -38,10 +38,10 @@ no_proxy =
 baseurl= <%= @baseurl %>
 
 # Server CA certificate location:
-ca_cert_dir = /etc/rhsm/ca/
+ca_cert_dir = <%= @ca_cert_dir %>
 
 # Default CA cert to use when generating yum repo configs:
-repo_ca_cert = <%= @repo_ca_cert %>
+repo_ca_cert = %(ca_cert_dir)s<%= @repo_ca_cert_filename %>
```

Comment:
is this line actually correct? It looks strange

Reply:
As mentioned earlier subscription-manager is Python based. The default setting is

```
repo_ca_cert = %(ca_cert_dir)sredhat-uep.pem
```

https://access.redhat.com/documentation/en-us/red_hat_subscription_management/1/html/rhsm/rhsm-config

Comparison of the conf-file and the resulting configuration

```
[root@rhel ~]# grep repo_ca_cert /etc/rhsm/rhsm.conf 
repo_ca_cert = %(ca_cert_dir)sredhat-uep.pem
[root@rhel ~]# subscription-manager config|grep repo_ca_cert
   repo_ca_cert = /etc/rhsm/ca/redhat-uep.pem
```

Reply:
I'm OK with this.